examples/basics/demo_custom_physics.py

```diff
@@ -0,0 +1,194 @@
+r"""
+Bring your own physics
+====================================================================================================
+
+This examples shows you how to use DeepInverse with your own physics.
+
+While DeepInverse offers a :ref:`large number of forward operators <physics>`,
+you can also bring your own forward operator for your specific imaging problem.
+
+DeepInverse's modular :ref:`physics framework <physics_intro>` makes this easy by letting you inherit useful methods from the most appropriate
+physics base class:
+
+* :class:`deepinv.physics.Physics` for non-linear operators;
+* :class:`deepinv.physics.LinearPhysics` for linear operators;
+* :class:`deepinv.physics.DecomposablePhysics` for linear operators with a closed-form singular value decomposition.
+
+.. seealso::
+
+    Often your physics can be modelled without much work. You could:
+
+    * Use an existing physics but with custom `params`, e.g. blur with a custom kernel, or MRI with a custom sampling pattern. See :ref:`parameter dependent operators <parameter-dependent-operators>`.
+    * Inherit from an existing physics but override or wrap a particular method e.g. :class:`deepinv.physics.MRI` :math:`\rightarrow` :class:`deepinv.physics.DynamicMRI`
+    * Define a new operator by :ref:`combining existing operators <physics_combining>`.
+
+In this example we will create a simple forward operator that converts RGB images to grayscale images.
```

Comment:
```suggestion
In this example we will create a simple forward operator from scratch that converts RGB images to grayscale images.
```

to emphasize that we create it without one of the options listed above